ipwb/util.py

```diff
@@ -336,3 +340,28 @@ def check_for_update(_):
         print("The installed version of ipwb is outdated.")
         print(f"* Installed: {current}\n* Latest:    {latest}")
         print("Please run `pip install --upgrade ipwb` to upgrade.")
+
+
+def is_wacz(path):
+    # TODO: add logic to check if wacz
+    # the py-wacz validator inherits many dependencies,
+    # so ad hoc here for now
+    return is_zipfile(path)
+
+
+def get_warc_paths_in_wacz(wacz_path):
+    with ZipFile(wacz_path) as z:
+        return [w for w in z.namelist() if w.startswith('archive/')]
+
+
+def extract_warcs_to_disk(wacz_path, warc_paths):
+    for warc in warc_paths:
+        with ZipFile(wacz_path) as z:
+            z.extract(warc)
+
+
+def extract_warcs_from_wacz(wacz_path):
```

Comment:
Would it not be easier to unpack the whole WACZ file (with force overwrite), ideally in a new temp folder, instead of extracting WARCs one by one? That way, the whole `archive/` folder can be added in the WARCs targets to scan from.

Reply:
Complete in [e4b9998](https://github.com/oduwsdl/ipwb/pull/770/commits/e4b9998f82f7cb75f63ed808ec5080c0e7449074)